documentation/howto/maven_basics.adoc

```diff
@@ -0,0 +1,147 @@
+= Basic Maven Configuration
+:page-layout: howto
+:page-tab: docs
+:page-status: green
+:experimental:
+:imagesdir: ./images
+
+In the context of application development, Maven allows you to extend your applications by referencing and utilizing resources from local and remote repositories. This article serves as an introduction to using Maven with the IDE and contains instructions for the following tasks:
```

Comment:
I think using the word "extend" for what Maven does is unfortunate. It is not 100% incorrect but it is a misleading term imo.

Maven is much more about providing standardized build system for your projects, and *one* part of this build system is that it allows you to fetch dependencies from one or multiple repositories.